packages/jest-transform/src/__tests__/__snapshots__/script_transformer.test.js.snap

```diff
@@ -126,6 +126,7 @@ cov_25u22311x4();
 cov_25u22311x4().s[0]++;
 module.exports = "banana";
 //# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbImJhbmFuYS5qcyJdLCJuYW1lcyI6WyJtb2R1bGUiLCJleHBvcnRzIl0sIm1hcHBpbmdzIjoiOzs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7QUFlWTs7Ozs7Ozs7OztBQWZaQSxNQUFNLENBQUNDLE9BQVAsR0FBaUIsUUFBakIiLCJzb3VyY2VzQ29udGVudCI6WyJtb2R1bGUuZXhwb3J0cyA9IFwiYmFuYW5hXCI7Il19
+//# sourceMappingURL=/cache/jest-transform-cache-test/5b/banana_5bf5b7017ea79e485ba49f739d279573.map
```

Comment:
@aleclarson both inline and map link here - seems wrong. I'm worried this might cause issues (similar to what they tackled here https://github.com/vuejs/vue-jest/pull/255)

Reply:
Interesting, so the `Runtime._instrumentFile` method appends an inline sourcemap. Good to know!

I updated my patch to avoid appending the `sourceMappingURL` when code coverage is enabled for a file. That should avoid any conflicts!